CMakeLists.txt

```diff
@@ -465,9 +494,35 @@ endif()
 # Start script
 IF (WIN32)
 	install(PROGRAMS "release/bin/rttr.bat" DESTINATION "${RTTR_BINDIR}")
+ELSEIF(APPLE)
+	install(PROGRAMS "release/bin/macos/rttr.command" DESTINATION "${RTTR_LIBDIR}")
```

Comment:
Although this might technically be the correct path, don't use `RTTR_LIBDIR` as that is the wrong path (name variable)
If we do want to hard-code the whole .app-structure, define variables `SET(RTTR_MACOS_DIR "./s25client.app/Contents/MacOS")` or even the parent dir: `SET(RTTR_MAC_CONTENT_DIR "./s25client.app/Contents")`. Because looking at this lines make me want to change them as they *look* wrong